runtime/themes/ayu_dark.toml

```diff
@@ -47,7 +51,7 @@
 "ui.text.info" = "foreground"
 "ui.virtual.whitespace" = "dark_gray"
 "ui.virtual.ruler" = { bg = "black" }
-"ui.virtual.inlay-hint" = { fg = "#e6b450", bg = "#302a20" }  # original bg #e6b45033
+"ui.virtual.inlay-hint" = { fg = "gray" }
```

Comment:
Also, I put orange previously because it looks like the rest of the ayu themes, as seen in vscode.

Yes, gray looks more like other themes but I find orange quite readable and nice too.

Reply:
Sure, the gray might be personal preference, I can overwrite that locally.